vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/finder/StepDefinitionFinder.java

```diff
@@ -99,6 +112,43 @@ public Collection<StepDefinition> find(IJavaProject root)
         return stepDefinitions;
     }
 
+    private List<StepDefinition> findLocalCompositeSteps(Path resourcesFolder) throws IOException
+    {
+        List<StepDefinition> composites = new ArrayList<>();
+
+        Files.walkFileTree(resourcesFolder, new SimpleFileVisitor<Path>()
+        {
+            @Override
+            public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)
+                throws IOException
+            {
+                String directoryName = dir.getFileName().toString();
+                return "properties".equals(directoryName) || "story".equals(directoryName)
```

Comment:
I wouldn't put such restrictions, it's hard to imagine what users can do despite the fact it doesn't have logical explanation